### PR TITLE
Custom system support

### PR DESCRIPTION
**Pull request by iwontgiveyoumyname**
I'm currently working on a custom system. I was working on SWS but I moved onto my own system, so it's work almost the same
I'm not to sure if I changed everything I was supposed to change, so if needed, I can still remove the request and create another one.

### CODE REVIEW

**Comment by trioderegion** [comment]
Has this been tested? I see syntax errors and missing files.

`"./systems/vaarfeu.js;`

**Comment by iwontgiveyoumyname** [comment]
Well to be honest, I have literaly no idea of what I'm supposed to change or even to do for it to working with my custom system, I have 0 experience with all of that stuff
I tried to ask to honeybadger on discord but I didn't get to know what exaclty I had to change for that sorry

**Comment by trioderegion** [comment]
> I tried to ask to honeybadger on discord but I didn't get to know what exaclty I had to change for that sorry

Hi Hi, itme, honeybadger. @trioderegion is my github username and I'm babysitting this module while kekilla is away.  I'm sorry I can't provide more guidance on this specific implementation, but duplicating the SWS handler should get you close.

File: scripts/helper.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import * as ose from "./systems/ose.js";
 import * as demonlord from "./systems/demonlord.js";
 import * as cyberpunk from "./systems/cyberpunk-red-core.js";
 import * as worldbuilding from "./systems/worldbuilding.js";
+import * as vaarfeu from "./systems/vaarfeu.js;
```
Comment (trioderegion) [inline scripts/helper.js:11]: syntax error

File: scripts/helper.js [hunk 2]
```diff
@@ -112,6 +113,9 @@ export class helper{
       case "worldbuilding" :
         if(settings.value("defaultmacro")) worldbuilding.register_helper();
         break;
+      case "vaarfeu" :
+        if(settings.value("defaultmacro"))  vaarfeu.register_helper();
```
Comment (trioderegion) [inline scripts/helper.js:117]: undefined symbol `vaarfeu`

File: scripts/helper.js [hunk 3]
```diff
@@ -183,6 +187,9 @@ export class helper{
       case "worldbuilding" :
         if(settings.value("charsheet")) return worldbuilding.sheetHooks();
         break;
+      case "vaarfeu" :
+        if(settings.value("charsheet")) return vaarfeu.sheetHooks();
```
Comment (trioderegion) [inline scripts/helper.js:191]: undefined symbol `vaarfeu`

**Review by trioderegion — CHANGES_REQUESTED** [review]
Please address noted changes.

File: scripts/helper.js (hunk 1 above)
Comment (trioderegion) [reply scripts/helper.js]: Addressed in update c8d5

**Pull request closed without merging**